docs/superpowers/plans/2026-05-13-species-selector-mobile.md

```
@@ -0,0 +1,806 @@
+    {sidebarOpen ? (
+      // ... existing expanded sidebar JSX — DO NOT CHANGE ...
+    ) : (
+      <CollapsedSidebarStrip
+        filters={filters}
+        onExpand={() => setSidebarOpen(true)}
+      />
+    )}
+  </div>
+)}
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)